<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="FeedbackForm_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Feedback Form</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="FirstAndLastName_grid_layout">
     <item row="1" column="0">
      <widget class="QLabel" name="FirstName_label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <fontweight>Thin</fontweight>
        </font>
       </property>
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="FirstName_line_edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="LastName_label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
         <fontweight>Thin</fontweight>
        </font>
       </property>
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="LastName_line_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Feedback_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Please provide your feedback</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>371</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="FeedbackRadioButtons_hbox_layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QRadioButton" name="Excellent_radio_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Excellent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="VeryGood_radio_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Very good</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Good_radio_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Good</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Average_radio_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Average</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Poor_radio_button">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Poor</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Reason_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>What is the main reason for your answer?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Reason_line_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Submit_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>49</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
